deliverables/README.md

```diff
@@ -2,13 +2,22 @@
 #### Required
 ##### Questions
 - What is a framework?
+-They are large bodies of prewritten code that you can add to your own code to solve certain problems. In a framework all the control flow is all ready defined, and there's a bunch of predefined white spots that you can fill out the your code.
+ 
```

Comment:
A framework adds boilerplate and structure to an application. It is like a foundation for a house.